src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 200px;
```

Comment:
The width of the card is set to 200px, which might not be responsive. Consider using relative units like percentages or viewport units for better responsiveness.

Reply:
thats the requirement